docs/en/quickstart/data_import_guide.md

```diff
@@ -0,0 +1,42 @@
+# Data Import Quickstart
+ 
+There are 2 versions of OpenMLDB: the standalone version and cluster version.
+- For standalone version, datasets are all saved in the memory. Only [`LOAD DATA`](../reference/sql/dml/LOAD_DATA_STATEMENT.md) can be used to import data in this mode.
+- For cluster version, datasets can be saved offline or online. Offline and online ends don't share the data.
+
+This tutorial will focus on the data import methods of cluster version.
+
+## Data Import Methods of Cluster Version
+
+### 1 Offline Import (`LOAD DATA`)
+
+- OpenMLDB doesn't have offline storage engine, but it requires user to specify the offline storage path, that is modifying the configuration option of taskmanager: `offline.data.prefix`. You can use storage medium like local directory, hdfs, s3 to configure.
+- There is only one way to import data offline: using [`LOAD DATA` command](../reference/sql/dml/LOAD_DATA_STATEMENT.md). Hard copy will be adopted as default.
+- OpenMLDB will copy the original data to the path of `offline.data.prefix` by default. The files of `csv` and `parquet` format are supported. 
+- `LOAD DATA` with a soft link is also supported, you can use the option `deep_copy=false` to configure. Only the storage path of the datasets will be saved in OpenMLDB in a soft link. Both the `csv` and `parquet` files are supported as well.
+
+
+```{note}
+If the offline path of the table is soft link, OpenMLDB doesn't support appending data to the table as it doesn't have write access to the files in the **soft link path**. You can overwrire the offline path of the table. If the path has been overwritten, the data in the original directory will not be removed, only the directory in the OpenMLDB will change.
+```
+
+### 2 Online Import
+
+The online modes of OpenMLDB cluster version provide online storage engine (saved in memory). Only **hard copy** can be used in online import.
+
+#### 2.1 `LOAD DATA`
+
+[`LOAD DATA` command](../reference/sql/dml/LOAD_DATA_STATEMENT.md) can be used in **Online Request** and **Online Preview** mode to load `csv` files and `parquet` files.
+
+#### 2.2 Stream
+
+Data can be loaded from `pulsar` as well, see [Pulsar Connector](../use_case/pulsar_connector_demo.md) for detail.
```

Comment:
also add Kafka and RocketMQ

Reply:
These three methods have been added.
but since there is no en version of rocketMQ doc, the link of it may not work

Reply:
> These three methods have been added. but since there is no en version of rocketMQ doc, the link of it may not work

you use the chinese doc first :-)